templates/zerver/emails/digest.subject.txt

```diff
@@ -1 +1 @@
-Zulip digest for {{ realm_name }}
+{% trans %}Zulip digest for{% endtrans %} {{ realm_name }}
```

Comment:
You need the `realm_name` part inside the translation tag here to support languages with another word order.